.github/CODEOWNERS

```diff
@@ -0,0 +1,238 @@
+/helmfile.d/charts/harbor/ @elastisys/goto-container-registry
+/helmfile.d/charts/hnc/ @elastisys/goto-security
+/helmfile.d/charts/ingress-nginx-probe-ingress/ @elastisys/goto-ingress-auth
+/helmfile.d/charts/issuers/ @elastisys/goto-ingress-auth
+/helmfile.d/charts/kube-state-metrics-extra-resources/ @elastisys/goto-monitoring-stack
+/helmfile.d/charts/kubeapi-metrics @elastisys/goto-monitoring-stack
+/helmfile.d/charts/kured-secrets/ @elastisys/goto-security
+/helmfile.d/charts/log-manager/ @elastisys/goto-logging-stack
+/helmfile.d/charts/networkpolicy/ @elastisys/goto-security
+/helmfile.d/charts/node-local-dns/ @elastisys/goto-kubernetes
+/helmfile.d/charts/opensearch/ @elastisys/goto-logging-stack
+/helmfile.d/charts/openstack-monitoring/ @elastisys/goto-kubernetes
+/helmfile.d/charts/prometheus-alerts/ @elastisys/goto-monitoring-stack
+/helmfile.d/charts/prometheus-servicemonitor/ @elastisys/goto-monitoring-stack
+/helmfile.d/charts/rclone/ @elastisys/goto-security
+/helmfile.d/charts/s3-exporter/ @elastisys/goto-monitoring-stack
+/helmfile.d/charts/tekton-pipelines/ @elastisys/goto-continuous-delivery
+/helmfile.d/charts/thanos/ @elastisys/goto-monitoring-stack
+/helmfile.d/charts/user-alertmanager/ @elastisys/goto-monitoring-stack
+/helmfile.d/charts/user-crds/ @elastisys/goto-security
+/helmfile.d/charts/user-rbac/ @elastisys/goto-security
+
+#
+# Skipped in helmfile.d/charts/
+# namespaces
+#
+
+#
+# helmfile.d/hooks/
+#
+/helmfile.d/hooks/dev-rbac/ @elastisys/goto-security
+/helmfile.d/hooks/fluentd/ @elastisys/goto-logging-stack
+
+#
+# helmfile.d/stacks/
+#
+
+/helmfile.d/stacks/calico.yaml @elastisys/goto-kubernetes
+/helmfile.d/stacks/cert-manager.yaml @elastisys/goto-ingress-auth
+/helmfile.d/stacks/dex.yaml @elastisys/goto-ingress-auth
+/helmfile.d/stacks/external-dns.yaml @elastisys/goto-ingress-auth
+/helmfile.d/stacks/falco.yaml @elastisys/goto-security
+/helmfile.d/stacks/fluentd.yaml @elastisys/goto-logging-stack
+/helmfile.d/stacks/gatekeeper.yaml @elastisys/goto-security
+/helmfile.d/stacks/hnc.yaml @elastisys/goto-security
+/helmfile.d/stacks/ingress-nginx.yaml @elastisys/goto-ingress-auth
+/helmfile.d/stacks/kured.yaml @elastisys/goto-security
+/helmfile.d/stacks/monitoring-grafana.yaml @elastisys/goto-monitoring-stack
+/helmfile.d/stacks/monitoring-prometheus.yaml @elastisys/goto-monitoring-stack
+/helmfile.d/stacks/monitoring.yaml @elastisys/goto-monitoring-stack
+/helmfile.d/stacks/opensearch.yaml @elastisys/goto-logging-stack
+/helmfile.d/stacks/rbac.yaml @elastisys/goto-security
+/helmfile.d/stacks/rclone.yaml @elastisys/goto-security
+/helmfile.d/stacks/system.yaml @elastisys/goto-security # node-local-dns here as well
+/helmfile.d/stacks/tekton.yaml @elastisys/goto-continuous-delivery
+/helmfile.d/stacks/thanos.yaml @elastisys/goto-monitoring-stack
+/helmfile.d/stacks/velero.yaml @elastisys/goto-security
+
+#
+# Skipped files in /helmfile.d/stacks/
+# local.yaml
```

Comment:
`goto-appdev-experience`?

Reply:
See my comment above about minio

Reply:
I still think that we should have some ownership to the contributor experience. But that might be QA then

Reply:
Yeah I think QA is closest at least. I'll add it to them for now.